src/core/components/dataList/dataListFilter/dataListFilterStatus.tsx

```diff
@@ -20,28 +23,35 @@ export const DataListFilterStatus: React.FC<IDataListFilterStatusProps> = () =>
     }
 
     return (
-        <p
-            className={classNames(
-                'px-3 text-sm font-normal leading-tight text-neutral-500 md:text-base',
-                { 'text-primary-400': isLoading || isInitialLoading },
-                { 'text-neutral-500': displayItemsCount },
-            )}
-        >
-            {displayItemsCount && (
-                <>
-                    <span className="text-neutral-800">{itemsCount} </span>
-                    <span>{entityLabel}</span>
-                </>
-            )}
-            {isInitialLoading && copy.dataListFilterStatus.loadingEntity(entityLabel)}
-            {isLoading && copy.dataListFilterStatus.filteringEntity(entityLabel)}
+        <div className="flex h-10 items-center justify-between">
```

Comment:
We don't probably need to set a fixed height here
```suggestion
        <div className="flex items-center justify-between">
```

Reply:
@cgero-eth problem with not setting height is the content on the left is pushed down as soon as state transitions to `isFiltered` and the UI jumps if that makes sense.